Gem/Code/Source/SampleComponentManager.cpp

```diff
@@ -1561,33 +1561,40 @@ namespace AtomSampleViewer
         m_rhiScene = RPI::Scene::CreateScene(sceneDesc);
         m_rhiScene->Activate();
 
-        RPI::RenderPipelineDescriptor pipelineDesc;
+        const bool xrSystemRegistered = (AZ::RPI::RPISystemInterface::Get()->GetXRSystem() != nullptr);
+        const bool createDefaultRenderPipeline = !xrSystemRegistered || r_EnableDefaultRenderPipelineOnXR;
```

Comment:
Hmm. This would mean that if the cvar is false (which it is by default) and you enable openxr by command line you will not get the default pipeline on pc. Maybe we should set the cvar to be true by default for pc but false for android.  Essentially pal-ify the cvar. 

Since we are using one cvar (r_EnableDefaultRenderPipelineOnXR) for pipeline creation and enable/disable we probably want it to be true for pc so that it atleast gets created. If the cvar was false by default you never create the pipeline and hence cant turn it on. Which is sort of why I was thinking that maybe we have 2 cvars. But pali-fying this cvar may work for now.

Reply:
I see the approach you're going with, but for the ASV sample I thought that wasn't needed so it's different for what I was shooting for. For the ASV sample this is only a CVAR that is either on or off at launch and that's it. You want something that can be dynamically turn the pipeline on or off on PC. I can change the code to do that, it'll be have to be done again for the launcher/editor, for ASV I was just trying to keep it simple.

Reply:
I've update the code so the cvar is only available in host platforms (windows, linux, mac) and it can be used at any time to enable/disable the render pipeline when xr system is available.